sdk/docs/pages/typescript/cryptography/keypairs.mdx

```diff
@@ -126,6 +126,22 @@ const exampleMnemonic = 'result crisp session latin ...';
 const keyPair = Ed25519Keypair.deriveKeypair(exampleMnemonic);
 ```
 
+## Deriving a `Keypair` from a Bech32 encoded secret key
+
+You can derive a `Keypair` by converting the 33-byte secret key encoded in Bech32 string to a 
+`Uint8Array` and passing it to the `fromSecretKey` method of a `Keypair` class.
+
+```typescript
+import { Ed25519Keypair } from '@mysten/sui.js/keypairs/ed25519';
+import { decodeSuiPrivateKey, encodeSuiPrivateKey } from '@mysten/sui.js/cryptography/keypair';
+
+const encoded = encodeSuiPrivateKey([59, 148, 11, 85, 134, 130, 61, 253, 2, 174, 59, 70, 27, 180, 51, 107, 94, 203, 174, 253, 102,39, 170, 146, 46, 252, 4, 143, 236, 12, 136, 28], 'ED25519');
+const {schema, secretKey} = decodeSuiPrivateKey(encoded);
+// use schema to choose the correct key pair
+const keypair = Ed25519Keypair.fromSecretKey(secretKey);
```

Comment:
We should really make `getSecretKey()` and `fromSecretKey()` match.  I think it would be reasonable to detect if the provided secretKey is a string starting with suiprivkey and decode it automatically here

Reply:
you mean, change the fromSecretKey to take in a string instead of a bytearray? the getSecretKey() does return the Bech32 string, fromSecretKey takes the raw 32 bytes and creates the keypair based on the concrete type

Reply:
Accepting both the bech32 string and the raw bytes should work, that will be backwards compatible, and still make it easy to re-create an exported keypair

Reply:
ok, we can address in a separate pr, here i just want to make sure we have docs on this